src/cache/MemoryCache.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {MemoryCache} from "./MemoryCache";
+
+describe("MemoryCache", () => {
+    it("is defined", () => {
+        expect(MemoryCache).toBeDefined();
+    });
+
+    const store = new MemoryCache();
+
+    describe("setItem()", () => {
+        it("is function", () => {
+            expect(typeof store.setItem).toBe("function");
+        });
+
+        it("accepts key, value and ttl; returns nothing", () => {
+            expect(store.setItem("key", "value", 5)).toBeUndefined();
+        });
+    });
+
+    describe("getItem()", () => {
+        it("is function", () => {
+            expect(typeof store.getItem).toBe("function");
+        });
+
+        it("returns the saved value", () => {
+            expect(store.getItem("key")).toBe("value");
```

Comment:
Normally no, we want to test each block separately with out any order constraints.